.github/workflows/opam.yml

```diff
@@ -49,7 +47,7 @@ jobs:
           key: opam-${{ matrix.os }}-${{ matrix.ocaml-compiler }}-${{ hashFiles('**.opam') }}
 
       - name: Use OCaml ${{ matrix.ocaml-compiler }}
-        uses: ocaml/setup-ocaml@v2
+        uses: ocaml/setup-ocaml@v2.0.15
```

Comment:
this should be the same as before

Reply:
I was trying to see if updating the action will fix the windows error :(